taichi/runtime/cpu/jit_cpu.cpp

```diff
@@ -289,11 +289,28 @@ void JITSessionCPU::global_optimize_module_cpu(llvm::Module *module) {
   module_pass_manager.add(llvm::createSeparateConstOffsetFromGEPPass(false));
   module_pass_manager.add(llvm::createEarlyCSEPass(true));
 
+  llvm::SmallString<8> outstr;
+  raw_svector_ostream ostream(outstr);
+  ostream.SetUnbuffered();
+  if (this->config_.print_kernel_nvptx) {
```

Comment:
lol shall we also update the doc to make sure devs understand `print_kernel_nvptx` also controls cpu asm? (in fact we should rename this hh